### PR TITLE
UI basic setup

### PR DESCRIPTION
**Pull request by dsp9049**

### CODE REVIEW

File: src/user_interface/home_page.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    QSizePolicy, QFileDialog
+
+
+class HomePage(QWidget):
+
+    def __init__(self):
+        super().__init__()
+        self.nav_play = QPushButton("Play")
+        self.nav_settings = QPushButton("Settings")
+        self.lambda_thing = None
```
Comment (cha2251) [inline src/user_interface/home_page.py:14]: What dis

File: src/user_interface/home_page.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        # button.setIcon(QIcon(r"images\play-solid.svg"))
+        # button.setIconSize(QSize(65, 65))
+        # button.setToolTip('This is an example button')
+        # button.clicked.connect(self.on_click)
+
+        # quit_button = QPushButton("Exit", self)
+
+        title = QLabel("RIT Player Piano")
+        title.setAlignment(Qt.AlignCenter)
+        # title.setAlignment(Qt.AlignTop)
```
Comment (cha2251) [inline src/user_interface/home_page.py:33]: Delete this

File: src/user_interface/main_page.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    def go_to_settings_page(self):
+        self.stackLayout.setCurrentIndex(2)
+
+    def update_playing_page_song(self, song_name):
+        self.play_page.set_song(song_name)
+        print("MAIN UPDATE SONG NAME")
+
+
+if __name__ == '__main__':
+    app = QApplication([])
```
Comment (cha2251) [inline src/user_interface/main_page.py:58]: Can we make this a fucntion in this class that gets called from main.py?
that way everything runs through `python -m src.common.main`

**Review by cha2251 — CHANGES_REQUESTED** [review]

**Comment by DavidTheFighter** [comment]
Also, the image files used in the UI are currently inside the source directory. This means if I run the program with the usual `python -m src.user_interface.main_page` they won't get picked up. Do we want to move these to the top-level directory?

File: src/user_interface/home_page.py (hunk 1 above)
Comment (dsp9049) [reply src/user_interface/home_page.py]: fixed in latest version

File: src/user_interface/home_page.py (hunk 2 above)
Comment (dsp9049) [reply src/user_interface/home_page.py]: deleted

**Review by cha2251 — APPROVED** [review]
lgtm

**Pull request merged**